src/nvidia-cuda/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+install_cuda=${INSTALL_CUDA:-"true"}
+install_cudnn=${INSTALL_CUDNN:-"true"}
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    exit 1
+fi
+
+# Download NVIDIA's keyring package
+ubuntu_version="$(lsb_release -sr | sed 's/\.//g')"
+keyring_repo="https://developer.download.nvidia.com/compute/cuda/repos/ubuntu$ubuntu_version/x86_64"
```

Comment:
Do we need to handle different architectures for ARM or anything else?

Reply:
Features here should work on both Ubuntu and Debian - most bases are actually debian rather than ubuntu in the wild.  Does the apt repo need to be different?

I'd also recommend doing `. /etc/release` and referencing $ID and $VERSION_COODENAME here to avoid the dependency on `lsb_release`.  That's not in most images by default.

Reply:
Thanks, done!